### PR TITLE
Accordions version 3 updates

### LINKED ISSUE #292: Update Accordions layout/design to v3
**Description**
This card is a part of a series of cards to update the latest style of the accordions to version 2. Below are the updated visual designs for reference:

accordions_ui_v3

### PR DESCRIPTION
**Pull request by juliaelman**
This pull request is for the updated designs for accordions. It is also downstream from PR #301 and utilizes the variables inside of that work.

Fixes #292

### CODE REVIEW

File: assets/_scss/components/_accordions.scss [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
+      background: url('../img/minus.png') no-repeat;
+      background-position: 96%;
+      background-size: 13px;
       color: $color-base;
       cursor: pointer;
       display: inline-block;
       font-family: $font-sans;
       margin: 0;
-      padding: 10px 15px 10px 35px;
+      padding: 15px 30px;
```
Comment (msecret) [inline assets/_scss/components/_accordions.scss]: I think these should be rem, or em units. Is there a specific reason they're px here?

File: assets/_scss/components/_accordions.scss [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
+      background-size: 13px;
     }
 
     button {
-      background: url('../img/arrow-down.png') no-repeat;
-      background-position: 15px;
-      border-bottom: 1px solid #757575;
+      background: url('../img/minus.png') no-repeat;
+      background-position: 96%;
+      background-size: 13px;
```
Comment (msecret) [inline assets/_scss/components/_accordions.scss]: What's 13px?

Comment (juliaelman) [inline assets/_scss/components/_accordions.scss]: It is for the plus/minus background image. I doubled the size and then specified here to make it a bit crisper on retina displays.

File: assets/_scss/components/_accordions.scss (hunk 1 above)
Comment (juliaelman) [inline assets/_scss/components/_accordions.scss]: They were originally px and only left them this way b/c that is what they were before :smile: 

Changed in ea1d123

**Comment by maya** [comment]
@juliaelman visually looks good to me, but I heard @mollieru may want a "bordered" version so check with her.

**Comment by juliaelman** [comment]
@maya yup! This work is currently blocked and is awaiting final designs //cc @angel @carodew 

See https://trello.com/c/SzcgRNlR/388-v2-accordions for more info.

**Comment by juliaelman** [comment]
@maya @msecret this pull request has been updated to add in the bordered accordion option. Not 100% in agreement with the approach, but it is inline with the current `*-bordered` naming convention as tables. Let me know what you all think and/or if there are any changes!

**Pull request merged**